virtac/atip_server.py

```diff
@@ -60,6 +60,7 @@ def __init__(
         feedback_csv=None,
         mirror_csv=None,
         tune_csv=None,
+        emit_calc=True
```

Comment:
You need to update any docstrings for these changes. It's missing for `ATIPServer`, at least.

Reply:
The trailing commas are getting lost as well. If you're using vscode, you can set up a pre-commit action that can run things like flake8, black etc.